packages/react-sdk/docusaurus/docs/React/03-ui-components/core/stream-call.mdx

```diff
@@ -32,13 +32,15 @@ export const App = () => {
 ### `call`
 
 | Type   |
-| ------ |
+|--------|
 | `Call` |
 
+This props internally provides the call instance to the [`StreamCallProvider`](https://github.com/GetStream/stream-video-js/blob/main/packages/react-bindings/src/contexts/StreamCallContext.tsx#L24) and thereby having it passed in the `StreamCall` component makes the call instance available to all of its child components.
```

Comment:
```suggestion
`StreamCall` instance propagated to the component's children as a part of `StreamCallContext`.
```

I would not add link to a specific line in source code. That is hard to maintain.